src/libs/actions/PersonalDetails.js

```diff
@@ -106,7 +115,7 @@ function fetchTimezone() {
         name: 'timeZone',
     })
         .then((data) => {
-            const timezone = lodashGet(data, 'nameValuePairs.timeZone.selected', 'America/Los_Angeles');
+            const timezone = lodashGet(data, 'nameValuePairs.timeZone', {automatic: true, selected: 'America/Los_Angeles'});
```

Comment:
Would be nice if we can fix the max-len warning here.

Reply:
Second time we have seen `America/Los_Angeles` maybe we can make that one a constant like `DEFAULT_TIME_ZONE` in `CONST`.